main.go

```
@@ -1,17 +1,33 @@
+
+	db.AutoMigrate(&models.Feature{})
+
+	r := chi.NewRouter()
+
+	r.Use(middleware.Logger)
+
+	r.Get("/", func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte("Hello World!"))
+	})
+
+	http.ListenAndServe(":3000", r)
```

golangci-lint (1.64.8)

Error return value of `http.ListenAndServe` is not checked

(errcheck)